_rules/focusable-no-keyboard-trap-standard-nav-a1b64e.md

```diff
@@ -29,7 +29,7 @@ This rule applies to any [HTML or SVG element][] that is [focusable][].
 
 For each target element, focus can cycle to the browser UI by using [standard keyboard navigation](#standard-keyboard-navigation).
```

Comment:
```suggestion
For each target element, focus can be moved away from that element using only [standard keyboard navigation](#standard-keyboard-navigation).
```

I would suggest a better adherence with SC ([2.1.2: No Keyboard Trap](https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap.html)). 
- "focus can cycle to the browser UI" is related to the entire flattened tabindex ordered focus navigation scope, not to the single target element;
- "focus can cycle to the browser UI" navigating on Safari with the TAB, focus moves from the last tabbable element to the body element, skipping the "Browser UI". On the contrary, navigating backward (SHIFT+TAB) focus can be moved to the address-bar. Anyway, the new proposal should avoid any confusion.

Reply:
I'm a bit reluctant to change the expectation in this PR 🤔 Especially since that changes its scope quite a bit.
Can you open an issue for that? And we'll take it in a separate PR.